libraries/api-extractor/src/DocElementParser.ts

```diff
@@ -142,67 +142,72 @@ export default class DocElementParser {
     }
 
     // Make sure there are no extra pipes
-    let pipeSplitContent: string[] = tokenItem.text.split('|');
-    pipeSplitContent = pipeSplitContent.map( value => {
+    const pipeSplitContent: string[] = tokenItem.text.split('|').map(value => {
       if (value) {
         return value.trim();
       }
     });
     if (pipeSplitContent.length > 2) {
       documentation.reportError('The {@link} tag contains more than one pipe character ("|")');
-      return;
+      return undefined;
     }
 
+    const addressPart: string = pipeSplitContent.length > 0 ? pipeSplitContent[0] : '';
```

Comment:
According to the docs, this can only happen if the split() delimiter itself is an empty string.

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)

---
In reply to: [122822139](https://github.com/Microsoft/web-build-tools/pull/240#discussion_r122822139) [](ancestors = 122822139)